src/hooks/QueryGraphQL.tsx

```diff
@@ -0,0 +1,22 @@
+import { gql } from '@apollo/client';
+
+export const GET_BOBA_LIST = gql`
```

Comment:
Change to something like
Get Boba List Schema

Reply:
updated